src/customJs/components/EmptyViewer.test.js

```diff
@@ -0,0 +1,45 @@
+import { React } from '../unlayer-react';
+import { render, screen } from '@testing-library/react';
+import { EmptyViewer } from './EmptyViewer';
+
+describe(EmptyViewer.name, () => {
+  const rest = {
+    isViewer: true,
+    toolInfo: {
+      icon: 'toolIconPath',
+      label: 'toolLabel',
+      name: 'toolName',
+    },
+  };
+
+  it('should render the icon', async () => {
+    render(<EmptyViewer values={undefined} {...rest} />);
```

Comment:
Related to [this](https://github.com/FromDoppler/unlayer-editor/pull/930/files#r1280600033).

```suggestion
    render(<EmptyViewer {...rest} />);
```